dev/dune

```diff
@@ -12,6 +12,8 @@
   (deps dune-dbg.in
         ../checker/coqchk.bc
         ../topbin/coqc_bin.bc
+        ../ide/coqide_main.bc
```

Comment:
This dependency could be a pain if debugging somewhere without gtk deps. Not sure how often that happens.

Reply:
The rationale here is that anyways to be a developer and submit a PR you need a system with GTK [otherwise you cannot test the full build], but I can refactor this if you want.

Reply:
>The rationale here is that anyways to be a developer and submit a PR you need a system with GTK [otherwise you cannot test the full build]

IMO it's fine to let CI do that (the vast majority of PRs don't touch coqide) but I'm OK with the dependency if it makes things easier.